clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStats.java

```diff
@@ -207,6 +211,17 @@ public IngestionStats(VeniceServerConfig serverConfig) {
     registerSensor(localMetricRepository, OFFSET_REGRESSION_DCR_ERROR, offsetRegressionDCRErrorSensor);
     registerSensor(localMetricRepository, TOMBSTONE_CREATION_DCR, tombstoneCreationDCRSensor);
     registerSensor(localMetricRepository, IDLE_TIME, idleTimeSensor);
+
+    transformerLatencySensor = new WritePathLatencySensor(localMetricRepository, METRIC_CONFIG, TRANSFORMER_LATENCY);
```

Comment:
I'm confused here.  Why are some of these lazy and some not?  When do these get registered?

Reply:
Some metrics are required to be registered by some unit/integration tests by asserting that their value is 0. Should I modify the tests to assert that the sensor be null instead?

The underlying sensor inside `WritePathLatencySensor` is registered as lazy, but when a record is called on it the sensor will be registered.

Reply:
It's ok to edit tests if the behavior no longer matches up.  I think we should be uniform here unless theres a valid reason to not do so.